engine/src/main/java/org/terasology/rendering/particles/components/ParticleSystemComponent.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2015 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.particles.components;
+
+import org.lwjgl.util.vector.Vector2f;
+import org.terasology.entitySystem.Component;
+import org.terasology.entitySystem.Owns;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.rendering.assets.texture.Texture;
+import org.terasology.rendering.particles.events.ParticleSystemUpdateEvent;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Represents a particle system. A particle system must have emitters in order to produce particles.
+ */
+public class ParticleSystemComponent implements Component {
+
+    // Reference to owner entity
```

Comment:
Yeah my bad, I only remembered how javadoc is supposed to be formatted 3/4 of the way through working on this, I'll go over all my other stuff and make sure it's documented.